tools/clang/test/HLSL/vector-or.hlsl

```diff
@@ -0,0 +1,10 @@
+// RUN: %clang_cc1 -HV 2021 -fsyntax-only -ffreestanding -verify %s
+
+RWStructuredBuffer<int3> rw;
+
+[numthreads(1, 1, 1)]
+void main()
+{
+// expected-error@+1 {{operands for short-circuiting logical binary operator must be scalar, for non-scalar types use 'or'}}
```

Comment:
Can the Fixit component be verified here?

Reply:
I don't think so. We could probably use a FileCheck test for it, but what I'd really like to do is get the rewriter plumbed through so that we could write FileCheck tests that verify the applied FixIt, but that's a bigger change.

Reply:
Does it make sense to add a FileCheck test now or leave it as is and address tests to verify applied Fixits in a later change?

Reply:
I think it is fine as it is